people.html

```diff
@@ -70,41 +69,41 @@ <h1>People</h1>
 
 <h2>Board of Directors</h2>
 <ul>
-<li>Anthony Anderberg, <a href="https://nuvera.net" target="_blank">Nuvera Communications</a>, 2017&ndash;2020</li>
-<li>Reid Fishler, <a href="https://he.net" target="_blank">Hurricane Electric</a>, 2018&ndash;2021</li>
-<li>Richard Laager, <a href="https://wiktel.com" target="_blank">Wikstrom Telephone Company (Wiktel)</a>, 2019&ndash;2022</li>
+<li>Anthony Anderberg, <a href="https://nuvera.net/" target="_blank">Nuvera Communications</a>, 2017&ndash;2020</li>
+<li>Reid Fishler, <a href="https://he.net/" target="_blank">Hurricane Electric</a>, 2018&ndash;2021</li>
+<li>Richard Laager, <a href="https://wiktel.com/" target="_blank">Wikstrom Telephone Company (Wiktel)</a>, 2019&ndash;2022</li>
 </ul>
 <p>Directors are elected by the membership, at the fall meeting, to staggered three-year terms. Vacancies are filled by the remaining directors.</p>
 
 <h2>Officers</h2>
 <ul>
-<li><a href="mailto:chiefmanager@micemn.net">Chief Manager</a> (and President): Richard Laager, <a href="https://wiktel.com" target="_blank">Wikstrom Telephone Company (Wiktel)</a></li>
+<li><a href="mailto:chiefmanager@micemn.net">Chief Manager</a> (and President): Richard Laager, <a href="https://wiktel.com/" target="_blank">Wikstrom Telephone Company (Wiktel)</a></li>
 <li><a href="mailto:treasurer@micemn.net">Treasurer</a>: Kayla Olsen</li>
 <li><a href="mailto:secretary@micemn.net">Secretary</a>: None (handled by Chief Manager)</li>
 </ul>
 <p>Officers are appointed by the board with indefinite terms. It is current practice that the Chief Manager be a director.</p>
 
 <h2>Peering Committee</h2>
 <ul>
-<li><a href="mailto:peering@micemn.net">Peering Coordinator</a>: Jason Hanke, <a href="http://southfront.io" target="_blank">South Front Networks</a></li>
+<li><a href="mailto:peering@micemn.net">Peering Coordinator</a>: Jason Hanke, <a href="https://southfront.io/" target="_blank">South Front Networks</a></li>
 </ul>
 <p>The peering committee is responsible for coordinating with new participants and maintaining our <a href="https://www.peeringdb.com/ix/446" target="_blank">PeeringDB entry</a>. Members of the peering committee are appointed by the board with indefinite terms.</p>
 
 <h2>Technical Committee</h2>
 <ul>
-<li>Anthony Anderberg, <a href="https://nuvera.net" target="_blank">Nuvera Communications</a></li>
-<li>Doug McIntyre, <a href="https://www.iphouse.com" target="_blank">ipHouse</a></li>
-<li>Jeremy Lumby, <a href="http://mnvoip.com" target="_blank">Minnesota VoIP</a></li>
-<li>Mike Horwath</li>
+<li>Anthony Anderberg, <a href="https://nuvera.net/" target="_blank">Nuvera Communications</a></li>
+<li>Doug McIntyre, <a href="https://www.iphouse.com/" target="_blank">ipHouse</a></li>
+<li>Jeremy Lumby, <a href="http://mnvoip.com/" target="_blank">Minnesota VoIP</a></li> <!-- Broken HTTPS. -->
+<li>Mike Horwath, <a href="https://www.trility.io/" target="_blank">Trility Consulting</a></li>
```

Comment:
Did Mike confirm this, or where did you get this?

Reply:
It's on Mike's public Twitter and LinkedIn pages but can easily be removed/reverted if you don't want his company linked. Presumably Mike is serving MICE in a personal capacity, not by his company.

Reply:
It's fine to leave it.